app/views/api/v0/organizations/users.json.jbuilder

```diff
@@ -0,0 +1,7 @@
+json.array! @users do |user|
+  json.type_of "org_user"
+  json.extract!(user, :name, :username, :twitter_username, :github_username)
+
+  json.website_url user.processed_website_url
+  json.profile_image Images::Profile.call(user.profile_image_url, length: 320)
```

Comment:
see my point about rendering `api/v0/views/show.json.jbuilder` here (you can use a common partial for the two)

Reply:
I created a shared partial and pushed it up